app/subscriber/src/features/filter-media/FilterMedia.tsx

```diff
@@ -4,8 +4,8 @@ import { DateFilter } from 'components/date-filter';
 import { ContentListActionBar } from 'components/tool-bar';
 import { IContentSearchResult } from 'features/utils/interfaces';
 import React from 'react';
-import { useContent } from 'store/hooks';
-import { generateQuery, IContentModel, Loading, Show } from 'tno-core';
+import { useContent, useLookup } from 'store/hooks';
```

Comment:
Do not use the `useLookup` for settings.  Use the `useSettings` hook so that we have consistent behaviour in all places.

Reply:
Thank you! I am ready for your review :)

Reply:
The PR still is not using the correct hook.

Reply:
sorry about that, somehow I lost my changes. I am applying again. I will let you know. Thank you!